smdx/smdx_00002.xml

```diff
@@ -11,7 +11,7 @@
         <symbol id="FULL">3</symbol>
         <symbol id="FAULT">4</symbol>
       </point>
-      <point id="StVnd" offset="4" type="enum16" />
+      <point id="StVnd" offset="4" type="enum16" /> <!-- this is an enum? It has no symbols -->
```

Comment:
This is one of those 'empty' enums where the vendor can specify custom "vendor specific" values.